templates.go

```diff
@@ -1114,6 +1114,7 @@ coreos:
       --proxy-mode=iptables \
       --logtostderr=true \
       --kubeconfig=/etc/kubernetes/config/proxy-kubeconfig.yml \
+      --conntrack-max-per-core 131072 \
```

Comment:
do we need that on master nodes?

Reply:
Let's have it on all nodes for homogeneity.

Reply:
exactly